### PR TITLE
Option validation

### PR DESCRIPTION
**Pull request by gwicke**

### CODE REVIEW

**Comment by gwicke** [comment]
@wikimedia/services

File: README.md [hunk 1]
```diff
@@ -115,6 +115,27 @@ the record will be removed from storage.
 Please note, that setting custom `_ttl` for individual rows is not allowed for tables which
 have a revision retention policy set, as table-wide and row-wide TTLs would conflict.
 
+
+# Options
```
Comment (d00rman) [inline README.md]: Would be nice to expand on this a bit, like what do the different option values stand for.

Comment (Pchelolo) [inline README.md]: +1

Comment (gwicke) [inline README.md]: The comments in the example are intended to convey this, but I agree that we could add more background / detail.

Comment (gwicke) [inline README.md]: Added more detail in https://github.com/wikimedia/restbase-mod-table-spec/pull/27.

**Pull request merged**